src/SFProvider.cc

```diff
@@ -1,12 +1,14 @@
 #include "TauAnalysisTools/TauTriggerSFs/interface/SFProvider.h"
-
+#include <boost/algorithm/string.hpp>
 #include <algorithm>
 #include <sstream>
+#include <iostream>
```

Comment:
Nothing from the iostream include is used, right? So we can remove the include from the include directives.